rafx-framework/src/nodes/macro_render_feature.rs

```diff
@@ -31,5 +44,75 @@ macro_rules! declare_render_feature {
                 stringify!($struct_name)
             }
         }
+
+        fn render_feature_index() -> RenderFeatureIndex {
+            $struct_name::feature_index()
+        }
+
+        fn render_feature_debug_name() -> &'static str {
+            $struct_name::feature_debug_name()
+        }
+    };
+}
+
+#[macro_export]
+macro_rules! declare_render_feature_mod {
```

Comment:
I would remove all the macros that do nothing but add use statements. (I actually think it's bad practice for a macro to "leak" use statements). Left an alternative in another comment.

Reply:
Also in general, we could consider having a prelude for the different crates or add pub use in key places so people can do use rafx::assets::prelude::* and get sensible imports.